EIPS/eip-7621.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 7621
+title: Increase calldata cost
+description: Increase calldata cost to decrease the maximum block size
+author: Toni Wahrstätter (@nerolation), Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/tbd/1234
```

Comment:
FYI: You can create a discussions topic on Eth Magicians before a number is issued.

Reply:
Thanks - created one.